benchmarks/benchmark_zoom.py

```diff
@@ -26,12 +26,39 @@ class ZoomSuite:
 
     @discard_arg(1)
     def setup(self, dtype):
-        self.image = np.random.randn(256, 256, 256).astype(dtype)
+        self.image_1d = np.random.randn(2**24).astype(dtype)
+        self.image_2d = np.random.randn(4096, 4096).astype(dtype)
+        self.image_3d = np.random.randn(256, 256, 256).astype(dtype)
+        self.image_4d = np.random.randn(64, 64, 64, 64).astype(dtype)
```

Comment:
why not use a separate parameter "ndim" for this?

Reply:
:fair_enough: